erpnext/selling/doctype/sales_order/sales_order.js

```diff
@@ -319,7 +319,7 @@ erpnext.selling.SalesOrderController = class SalesOrderController extends erpnex
 						title: __('Select Items to Manufacture'),
 						fields: fields,
 						primary_action: function() {
-							var data = d.get_values();
+							var data = d.fields_dict.items.grid.get_selected_children();
```

Comment:
@s-aga-r check the failing test. The test is expecting the previous type of input data. 

For sake of backward compatibility and passing the tests, you should change the data format here so it matches the previous one. That would be `{"items": ...}`